app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -18,23 +22,21 @@ export const DosesDescription = ({
   dosesPercent,
   children = null,
 }: DosesDescriptionProps) => (
-  <div className="flex flex-row justify-start items-center">
-    <dt className="w-32 flex items-center">
+  <div className="description-definition">
+    <dt className="term">
       {hasMetTarget && (
-        <div className="w-4 h-4 mr-2 flex items-center justify-center rounded-full check">
-          <CheckIcon className="h-4 w-4" />
+        <div className="check">
+          <CheckIcon />
         </div>
       )}
       <div
-        className={cx("data-text", {
-          ["data-text-complete"]: hasMetTarget,
-        })}
+        className={cx("data-text", { ["data-text-complete"]: hasMetTarget })}
```

Comment:
Discussed briefly with Finn - these classes may move or change, but it's unclear atm whether they belong to this component or not. Leaving them out for now!